README.md

```diff
@@ -2,92 +2,133 @@
 
 ## Introduction
 
-The "Portal Network" is an in progess effort to enable lightweight protocol access to resource constrained devices.  The term *"portal"* is used to indicate that these networks provide a *view* into the protocol but are not critical to the operation of the core Ethereum protocol.
+The Portal Network is an in progess effort to enable lightweight protocol access to resource constrained devices.  The term *"portal"* is used to indicate that these networks provide a *view* into the protocol but are not critical to the operation of the core Ethereum protocol.
```

Comment:
*nit*: "enable access to devices" sounds like we're enabling people to access lightweight devices. -> "to enable lightweight protocol access by resource constrained devices".